helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "workload_cluster"
+  name: systemd.workload-cluster.rules
+spec:
+  groups:
+  - name: systemd
+    rules:
+    - alert: WorkloadClusterSystemdUnitFailed
+      annotations:
+        description: '{{`Systemd unit {{ $labels.name }} is failed on {{ $labels.instance }}.`}}'
+      expr: node_systemd_unit_state{cluster_type="workload_cluster",name!~"debug-tools.service|k8s-addons.service|sshd.+service|systemd-coredump.+service|systemd-networkd.service|update-engine-stub.service",state="failed"} == 1
+      for: 5m
+      labels:
+        area: kaas
+        severity: notify
+        team: ludacris
+        topic: infrastructure
+    - alert: WorkloadClusterCriticalSystemdUnitFailed
+      annotations:
+        description: '{{`Critical systemd unit {{ $labels.name }} is failed on {{ $labels.instance }}.`}}'
+        opsrecipe: critical-systemd-unit-failed/
+      expr: node_systemd_unit_state{cluster_type="workload_cluster",name=~"k8s-addons.service|systemd-networkd.service", state="failed"} == 1
+      for: 5m
+      labels:
+        area: kaas
+        severity: page
+        team: ludacris
+        topic: infrastructure
+    - alert: WorkloadClusterDisabledSystemdUnitActive
+      annotations:
+        description: '{{`Disabled Systemd unit {{ $labels.name }} is active on {{ $labels.ip }}.`}}'
+        opsrecipe: disabled-systemd-unit-active/
+      expr: node_systemd_unit_state{cluster_type="workload_cluster",name=~"update-engine-stub.service",state="active"} == 1
```

Comment:
Removed `locksmithd.service`
Changed `update-engine.service` to `update-engine-stub.service`
Is this correct @giantswarm/team-ludacris ?


Original expr
```
node_systemd_unit_state{cluster_type="workload_cluster",name=~"locksmithd.service|update-engine.service", state="active"} == 1
```

Reply:
Let's please don't change these expressions right now. locksmithd needs to be disabled for sure

Reply:
I think changing any unit names in a follow up PR makes more sense to me.

Reply:
Ok, I'll revert this to the original expression and leave this for later then.